content/sigs/cloud-native/index.adoc

```diff
@@ -16,36 +16,24 @@ tags:
 - configuration-as-code
 - jcasc
 leads:
-- "carlossg"
-- "markyjackson-taulia"
+- "marckk"
 participants:
-- "jglick"
+- "garethjevans"
+- "waveywaves"
+- "salaboy"
+- "jstrachan"
 - "oleg_nenashev"
-- "tracymiranda"
-- "ewelinawilkosz"
-- "ndeloof"
 - "olblak"
-- "svanoort"
-- "leonardo-rossetti"
 - "batmat"
-- "vic-iglesias"
-- "evan-brown"
-- "lnewman"
-- "linuxsuren"
-- "davidcurrie"
-- "deepy"
-- "dominicgunn"
-- "runzexia"
 
 organizations:
 - name: "CloudBees"
   github: "cloudbees"
-- name: "Google Cloud Platform"
-  github: "GoogleCloudPlatform"
-- name: "Microsoft Azure"
-  github: "azure"
+- name: "Red Hat"
+  github: "Red Hat"
 
 links:
+  https://github.com/jenkinsci/sig-cloud-native/blob/main/meetings.md[Cloud Native SIG repo where we keep all meeting notes and documents]
```

Comment:
```suggestion
```

It is a metadata YAML, there is already a link to meetings below

Reply:
It's a link to the GitHub repo where we keep all meeting notes and documents for the Cloud Native SIG.
And it contains current, relevant information.

GitHub is more public and accessible than a google doc (in many parts of the world).